Assets/Scripts/Missions/MissionKitchen1.cs

```diff
@@ -145,7 +145,10 @@ private void SpawnEnemies(List<MissionEnemy> enemies)
                     case MissionEnemy.EnemyType.Patroller:
                         Patroller patrol = enemyComponent as Patroller;
                         patrol.SetPoints(enemy.waypoints.Select(waypoint => waypoint.position).ToList());
-                        patrol.InitializeResponderParameters(enemy.startResponsePoint, enemy.wanderBounds.position, enemy.wanderBounds.radius);
+                        if (enemy.startResponsePoint != enemy.spawnPosition)
```

Comment:
What's going on here? I don't really get why we are doing this check.
Maybe add a comment in the code here to explain?

Reply:
Yeah will do, it's because your mission system tries to assign every nurse with an initial response point, even when not needed (e.g. in the case of the first patrolling nurse in the kitchen).

For the nurses who are spawned to actually respond to something (the dentures), I want to set their state to "Responding" to correctly use that behavior & display the question mark. Rather than create extra nurse subtypes for now, I just made it so having a responsePoint == spawnPoint mean that you're not actually responding once you spawn in.

Reply:
I see, that makes sense! Maybe we can just add a bool flag in the `MissionEnemy` class instead to indicate if we want to set the response point for this enemy? Under `Generic Enemy Settings`?  Seems more clear than checking `enemy.startResponsePoint != enemy.spawnPosition`.

Reply:
Sounds good, I'll add that in